### PR TITLE
# Initializing List Structure

### PR DESCRIPTION
**Pull request by direakanbi**
# TO-DO LIST PROJECT (WITH WEBPACK)
1. Initialized WebPack bundler and all the necessary plugins.
2. Added linters.
3. Rendered lists dynamically to the main page with OOP in JavaScript.
4. Added styling in relation to the given template.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```
Comment (SajjadAhmad14) [inline dist/index.html]: Please add a professional README as it's the first point of interaction with the project user.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+const list = document.querySelector('#todo-data');
+
+const data = [
+  {
+    description: 'Go swimming',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Create an animated puppet',
```
Comment (SajjadAhmad14) [inline src/index.js]: Please do not implement all the functionality of the list here in this file as it is the entry file for the webpack. You may create another file and import that file here as a module.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Dire Akanbi, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
PR has a title and summary. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by EricMbouwe — APPROVED** [review]
### Hi @direakanbi, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**